Sources/Foundation/FileHandle.swift

```diff
@@ -25,10 +25,10 @@ fileprivate let _close = Glibc.close(_:)
 #endif
 
 #if canImport(WinSDK)
-// We used to get the copy that was re-exported by CoreFoundation
+// We used to get the alias that was re-exported by CoreFoundation
 // but we want to explicitly depend on its types in this file,
 // so we need to make sure Swift doesn't think it's @_implementationOnly.
-import WinSDK
+public typealias HANDLE = UnsafeMutableRawPointer
```

Comment:
(See comments on increasing exported surface elsewhere.)

Reply:
`HANDLE` is already `public`ly exported in `FileHandle.swift`, so it doesn’t actually increase any of the exported interface.

However, I got to change a bit to remove the explicit `typealias` here. Once the `public` exportation is removed, `HANDLE` should not be exported any more.